docs_src/OVMS/quick_pipelinerun.md

```diff
@@ -0,0 +1,136 @@
+# Quick Start Guide to Run Pipeline
+
+## Prerequisites
+Before running, [set up the pipeline](./pipelinesetup.md).
+
+## Running OVMS with the camera simulator
+
+### Start the Camera Simulator
+
+1. Download the video files to the **sample-media** directory:
+    ```bash
+    cd benchmark-scripts;
+    ./download_sample_videos.sh;
+    cd ..;
+    ```
+
+    !!! Example "Example - Specify Resolution and Framerate"
+        This example downloads a sample video for 1080p@15fps.
+        ```bash
+        cd benchmark-scripts;
+        ./download_sample_videos.sh 1920 1080 15;
+        cd ..;
+        ```
+
+    !!! Note
+        Only AVC encoded files are supported.
+
+2. After the video files are downloaded to the **sample-media** folder, start the camera simulator:
+    ```bash
+    make run-camera-simulator
+    ```
+
+3. Wait for few seconds, and then check if the camera-simulator containers are running:
```

Comment:
we could add portainer screenshots here, no need for terminal commands :-)
not required for this PR, may be in the next ones.

Reply:
@NeethuES-intel  Portainer is not a prerequisites. it is a bonus if have

Reply:
@jim-wang-intel yes, definitely. Feel its better from UX perspective, doesn't need much docker command knowledge 😃

Reply:
We should add Portainer as its own feature so that we can customize the port. That might be nice to add once we have the docker-compose so that we can create a compose file for Portainer.